### PR TITLE
Write initial readme

### PR DESCRIPTION
**Pull request by cowboy-bebug**
Just an initial readme to get started.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+#### 2. Install Rust and set up Rust environment
+
+Install Rust on your machine through [here](https://rustup.rs/), and the following rust version and toolchains.
+```
+$ cargo --version
+$ rustup install nightly-2019-12-19
+$ rustup default nightly-2019-12-19
+$ rustup target add --toolchain=nightly-2019-12-19 wasm32-unknown-unknown
+$ rustup target add --toolchain=nightly-2019-12-19 x86_64-unknown-linux-musl
```
Comment (jordy25519) [inline README.md]: I think this is all that's required, if rustup installs stable already.

```bash
# required for compiling the "runtime"
rustup install nightly
rustup target add --toolchain=nightly wasm32-unknown-unknown
```

This one shouldn't be required
`rustup target add --toolchain=nightly-2019-12-19 x86_64-unknown-linux-musl`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+#### 3. Build the node binary and run
+
+Then clone the repo, build the binary and run it.
+```
+$ git clone https://github.com/cennznet/cennznet.git
+$ cd cennznet
+$ cargo build --release
+$ cargo run --release [TODO: check params]
```
Comment (jordy25519) [inline README.md]: if we do `cargo build --release` should probably say to do `./target/release/cennznet`

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+3. format your code using `cargo fmt`
+4. build and test your code locally with `cargo build` and `cargo test --all`
+5. create a pull request merging from your fork into our repo
+
+## Pull request format
+
+When creating a PR, please make sure to describe what the PR is about and provide us with any relevant information.
+Also make sure that *any of* the following questions are answered in the PR description:
+
+- what feature is the code adding?
```
Comment (jordy25519) [inline docs/CONTRIBUTING.md]: We have a good pull_request template file in Plug which we could re-use some points here e.g. bumping runtime version etc.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+#### 3. Build the node binary and run
+
+Then clone the repo, build the binary and run it.
+```
+$ git clone https://github.com/cennznet/cennznet.git
+$ cd cennznet
+$ cargo build --release
+$ ./target/release/cennznet [TODO: check params]
```
Comment (jordy25519) [inline README.md:52]: Could update this like so
```
./target/release/cennznet --help

# start a validator node for development
./target/release/cennznet --dev
```

**Review by hoani — APPROVED** [review]
LGTM

File: README.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+## Running a CENNZnet node
+
+There are a number of ways to run a CENNZnet node. Please choose one that suits best for your interest.
+
+### Using Docker
+
+Make sure Docker is installed and running on your machine.
+```
+$ docker pull cennznet/cennznet:1.0.0
+$ docker run -it cennznet/cennznet:1.0.0 [TODO: check params]
```
Comment (jordy25519) [inline README.md:15]: `pull` is handled automatically these days when trying to run.
Let's set this to the latest available image `1.0.0-rc2`

```
# Start a local validator on a development chain
docker run \
    -p 9933:9933 -p 9944:9944 \
    cennznet/cennznet:1.0.0-rc2 --dev --unsafe-ws-external --unsafe-rpc-external
```

**Pull request merged**